### PR TITLE
ref(onboarding-wizard): Fix skip task and update styling

### PR DESCRIPTION
**Pull request by maheskett**
#### Fix
- Clicking skip unmounted the evt target, so `Sidebar` `documentClickHandler` considered the click outside of the onboarding task bar and hid the panel
#### Refactor
- `createReactClass` to extend React.Component
- .less styles to styled components

#### Hover over task: Add a second platform
Screen Shot 2019-05-16 at 5 45 33 PM

#### Clicked skip on task: Add a second platform
Screen Shot 2019-05-16 at 6 14 18 PM

### CODE REVIEW

File: src/sentry/static/sentry/app/components/onboardingWizard/confirmation.jsx [hunk 1]
```diff
@@ -14,18 +22,39 @@ class Confirmation extends React.Component {
...
-      <div className="ob-confirmation" onClick={this.props.dismiss}>
-        <h3>{t('Need help?')}</h3>
-        <p>
-          <a href="mailto:support@sentry.io?subject=Help with onboarding">
-            {t('Ask us!')}
-          </a>{' '}
+      <Container hide={hide} onClick={dismiss}>
+        <Header>{t('Want help?')}</Header>
+        <div>
+          <Button priority="link" to={`/settings/${organization.slug}/support/`}>
```
Comment (EvanPurkhiser) [inline src/sentry/static/sentry/app/components/onboardingWizard/confirmation.jsx]: This is great too!!

File: src/sentry/static/sentry/app/components/onboardingWizard/confirmation.jsx [hunk 2]
```diff
@@ -14,18 +22,39 @@ class Confirmation extends React.Component {
...
+  display: ${p => {
+    return p.hide ? 'none' : 'flex';
+  }};
+  bottom: 0px;
+  align-items: center;
+  flex-direction: column;
+  justify-content: center;
+  background: rgba(255, 255, 255, 0.65);
+  width: 100%;
+  left: 0px;
```
Comment (EvanPurkhiser) [inline src/sentry/static/sentry/app/components/onboardingWizard/confirmation.jsx]: nit: can you move the `top`, `left`, bottom` attributes all close to each other?

File: src/sentry/static/sentry/app/components/onboardingWizard/toDoItem.jsx [hunk 3]
```diff
@@ -119,49 +118,142 @@ const TodoItem = createReactClass({
...
+  skipped: ['icon-close', 'borderDark'],
+  pending: ['icon-ellipsis', 'borderDark'],
+  complete: ['icon-checkmark-sm', 'green'],
+};
+
+const IndicatorIcon = styled(({status, ...props}) => (
+  <InlineSvg {...props} src={indicatorStyles[status][0]} />
+))`
+  position: relative;
+  color: ${p => {
```
Comment (EvanPurkhiser) [inline src/sentry/static/sentry/app/components/onboardingWizard/toDoItem.jsx]: Can just be `color: ${p => p.theme[indicatorStyles[p.status][1]]};` (just single lining the function body)

**Review by evanpurkhiser — APPROVED** [review]
Awesome work. This is great!

File: src/sentry/static/sentry/app/components/onboardingWizard/toDoItem.jsx [hunk 4]
```diff
@@ -119,49 +118,140 @@ const TodoItem = createReactClass({
...
+  ${p =>
+    p.status === 'skipped' &&
+    css`
+      color: ${p.theme.gray2};
+      text-decoration: line-through;
+    `}
+`;
+
+const SkipButton = styled(Button)`
+  display: ${p => p.hide && 'none'};
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/onboardingWizard/toDoItem.jsx]: How does emotion handle this when `hide` is false?

Comment (maheskett) [reply src/sentry/static/sentry/app/components/onboardingWizard/toDoItem.jsx]: Hm... would this be better?

```
${p =>
    p.hide &&
    css`
      display: none;
    `}
```

Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/components/onboardingWizard/toDoItem.jsx]: afaik It just drops the prop @billyvg

Comment (billyvg) [reply src/sentry/static/sentry/app/components/onboardingWizard/toDoItem.jsx]: Nope I literally don’t know how emotion handles it @maheskett

**Pull request merged**